english/Full Metal Alchemist/fma.md

```diff
@@ -0,0 +1,3 @@
+Fullmetal Alchemist (Japanese: 鋼の錬金術師 Hepburn: Hagane no Renkinjutsushi?, lit. "Alchemist of Steel") is a Japanese manga series written and illustrated by Hiromu Arakawa. It was serialized in Square Enix's Monthly Shōnen Gangan magazine between August 2001 and June 2010; the publisher later collected the individual chapters into twenty-seven tankōbon volumes. The world of Fullmetal Alchemist is styled after the European Industrial Revolution. Set in a fictional universe in which alchemy is one of the most advanced scientific techniques, the story follows the Elric brothers Edward and Alphonse, who are searching for a philosopher's stone to restore their bodies after a failed attempt to bring their mother back to life using alchemy.
```

Comment:
This line is longer than 120 characters. Could you break it up into multiple lines? That way it will be easier to read the raw files or view diffs on the command line.  No need to add blank lines in between, just press enter somewhere in the middle of the line.